website/app/templates/show.hbs

```diff
@@ -1,4 +1,7 @@
 <HeadLayout />
+
+{{page-title this.title}}
```

Comment:
😱 🤯 😍

Reply:
i think this is where we could do what @MelSumner is suggesting and somehow buildup `title` to contain the category _and_ the title.